source/assets/javascripts/cookie_helper.js

```diff
@@ -0,0 +1,18 @@
+// Used to set, get and check for cookie consent
+function checkForCookies(name) {
+  return !!getCookie(name);
+}
+
+function getCookie(name) {
+  let cookieArray = document.cookie.split(';');
+  return cookieArray.find(cookieCrumb => cookieCrumb.substring(name.length, 0) === name).split('=')[1];
```

Comment:
Split won't work on the function like this in this case. Also, find isn't supported in IE8 - I shouldn't have used it here without a polyfill: 

To get the desired effect:
function getCookie(cname) {
  var name = cname + "="
  var decodedCookie = decodeURIComponent(document.cookie)
  var cookieArray = decodedCookie.split(';')

  for(var i = 0; i < cookieArray.length; i++) {
    var cookie = cookieArray[i];
    while (cookie.charAt(0) == ' ') {
      cookie = cookie.substring(1);
    }
    if (cookie.indexOf(name) == 0) {
      return cookie.substring(name.length, cookie.length);
    }
  }
  return "";
}